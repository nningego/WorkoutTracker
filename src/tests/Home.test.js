import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Home from '../Home';

describe('<Home/>', () => {
  test('test', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Home/>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  })
});
