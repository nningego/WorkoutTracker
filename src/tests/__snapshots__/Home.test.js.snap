// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Track workouts here
  </Text>
  <Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    Version 1.0 + 1
  </Text>
</View>
`;
